test/app-tap/uri.test.lua

```diff
@@ -41,9 +41,88 @@ local function test_parse(test)
     test:is(u.host, '127.0.0.1', 'host')
     test:is(u.ipv4, '127.0.0.1', 'ipv4')
 
+-- {{
+
+    -- The literal format of an IPv6 address can contain the following
+    -- characters [0-9;a-f;A-F]. It was discovered in that the
+    -- uri does not support the characters A-F, although it should
+    -- according to the RFC 2732.
```

Comment:
Nit: I would move examples from RFC 2732 to its own test case.

Reply:
Done